client/js/verification-portal/left-side-nav.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react'
+import injectSheet from 'react-jss'
+
+import LogoutButton from '../authentication/logout-button'
+import GetResourceToVerifyButton from './components/get-resource-to-verify-button'
+
+class LeftSideNav extends React.Component {
```

Comment:
If you only have a render method, this doesn't need to be a class. You can just do:
```
const LeftSideNav = () => {
     const { classes, changeFieldValue } = this.props
     return (
          <div..........
     )
}
```